### PR TITLE
Fill packetstructure

### PR DESCRIPTION
**Pull request by justinjoy**
This set of commits is partially filled packet structure section.

### CODE REVIEW

File: draft-sharabayko-srt.md [hunk 1]
```diff
@@ -290,13 +392,35 @@ Receiving Rate (32 bits):
 
 ### NAK (Loss Report)
 
+Negative acknowledgement control packets are used to signal failed
+data packet deliveries.
+
+Message Type: 
+: The type value of NAK control packet is "3".
```
Comment (maxsharabayko) [inline draft-sharabayko-srt.md]: Message Type -> Control Type

File: draft-sharabayko-srt.md [hunk 2]
```diff
@@ -290,13 +392,35 @@ Receiving Rate (32 bits):
...
 
 
 ### Congestion Warning
 
 ### Shutdown
 
+Shutdown control packets are used to initiate the closing of an SRT connection.
 
+Message Type: 
+: The type value of Shutdown control packet is "5".
```
Comment (maxsharabayko) [inline draft-sharabayko-srt.md]: Control Type

File: draft-sharabayko-srt.md [hunk 3]
```diff
@@ -426,4 +550,33 @@ TODO acknowledge.
...
+For any single packet sequence number or two consecutive sequence numbers, 
+it uses the original sequence number in the field. The first bit 
+must start with "0".
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|0|                   Sequence Number                           |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```
Comment (maxsharabayko) [inline draft-sharabayko-srt.md]: > or two consecutive sequence numbers

The first figure is for one sequence number. There might be confusion from the first paragraph.

Maybe this way:
_The receiver notifies the sender about the lost packets using the NAK packets. The NAK packet contains a list of sequence numbers of lost packets. 
To report a loss of one sequence, the first bit must start with "0"..._

Comment (justinjoy) [reply draft-sharabayko-srt.md]: Just removed `or two consecutive sequence numbers`. Logically, I think we don't need to care about the case as outstanding. 

Another point is that, as I mentioned in a commit message(https://github.com/Haivision/srt-rfc/pull/6/commits/c1c4de61544d400bdbc825b7d5e1cbc86785fb6e),  the list coding is a very general way so I tend not to mention 'Lost' or 'Loss'  in the appendix section.

**Pull request merged**